src/editor/profiler_ui.cpp

```diff
@@ -378,8 +378,8 @@ struct ProfilerUIImpl : public ProfilerUI
 		Lumix::Array<float> m_frames;
 		struct Hit
 		{
-			Lumix::uint64 start;
```

Comment:
pointer to hit contents is used [here](https://github.com/MAnyKey/LumixEngine/blob/linux-port-9/src/editor/profiler_ui.cpp#L692) btw I just realized that better decision is to allocate temporary array right there and pass pointer to it.

I changed definition of `uint64` to `unsigned long` on 64-bit Linux so that it is compatible with `cstdint` (otherwise there is bunch of error messages from usages of bgfx which uses `cstdint`'s ints.